gradle/gradle-mvn-push.gradle

```diff
@@ -42,128 +42,29 @@ def getRepositoryPassword() {
     return hasProperty('SONATYPE_NEXUS_PASSWORD') ? SONATYPE_NEXUS_PASSWORD : ""
 }
 
-def configurePom(pom) {
-    pom.groupId = GROUP
-    pom.artifactId = POM_ARTIFACT_ID
-    pom.version = VERSION_NAME
-
-    pom.project {
-        name POM_NAME
-        packaging POM_PACKAGING
-        description POM_DESCRIPTION
-        url POM_URL
-
-        scm {
-            url POM_SCM_URL
-            connection POM_SCM_CONNECTION
-            developerConnection POM_SCM_DEV_CONNECTION
-        }
-
-        licenses {
-            license {
-                name POM_LICENCE_NAME
-                url POM_LICENCE_URL
-                distribution POM_LICENCE_DIST
-            }
-        }
-
-        developers {
-            developer {
-                id POM_DEVELOPER_ID
-                name POM_DEVELOPER_NAME
+publishing {
```

Comment:
Not sure if this actually works 100%, I tested by publishing locally, and saw all three jars created as expected. But I'd also like to test by publishing with sonatype credentials.